SU-SWS/saa_victoria_subtheme
# Adapt 366  fixing style inheritance

# ALMOST READY FOR REVIEW

# Summary
- Converted scss to not use @apply lines so it compiles without including the problematic tailwind version.

# Review By (Date)
- 6/16

# Urgency
- important for tasks going forward

# Steps to Test

1. pull down the branch
1. check that main styles look correct for banner, cards, buttons, links
2. also run npm run build and make sure that it compiles

# Affected Projects or Products
- just this saa subtheme

# Associated Issues and/or People
- ADAPT-366

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
